samples/WebApiContrib.Core.Formatters.CsvFormatterSample/Controllers/CsvTestController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using WebApiContrib.Core.Formatters.CsvFormatterSample.Model;
+
+namespace WebApiContrib.Core.Formatters.CsvFormatterSample.Controllers

```

Comment:
we can later move away from that, but for start should be better, especially for the less complicated features